content/tutorial/tutorial.md

```diff
@@ -384,31 +389,31 @@ When we fill the board in later, the `this.state.squares` array will look someth
 ]
 ```
 
-The Board's `renderSquare` method currently looks like this:
+वर्तमान में Board का `renderSquare` मेथड इस तरह दिखता है:
 
 ```javascript
   renderSquare(i) {
     return <Square value={i} />;
   }
 ```
 
-In the beginning, we [passed the `value` prop down](#passing-data-through-props) from the Board to show numbers from 0 to 8 in every Square. In a different previous step, we replaced the numbers with an "X" mark [determined by Square's own state](#making-an-interactive-component). This is why Square currently ignores the `value` prop passed to it by the Board.
+शुरुआत में, हमने प्रत्येक Square में 0 से 8 तक संख्या दिखाने के लिए Board से [`value` prop को नीचे पास](#passing-data-through-props) किया। एक अलग पिछले चरण में, हमने संख्याओं को एक "X" चिह्न के साथ बदल दिया था [वर्ग के अपने state द्वारा निर्धारित](#making-an-interactive-component)। यही कारण है कि वर्तमान में Square ने Board द्वारा इसके लिए दिए गए `value` prop को अनदेखा कर दिया है।
 
-We will now use the prop passing mechanism again. We will modify the Board to instruct each individual Square about its current value (`'X'`, `'O'`, or `null`). We have already defined the `squares` array in the Board's constructor, and we will modify the Board's `renderSquare` method to read from it:
+हम अब props पासिंग मैकेनिज्म का फिर से उपयोग करेंगे। हम प्रत्येक व्यक्तिगत Square को उसके वर्तमान वैल्यू के बारे में निर्देश देने के लिए Board को संशोधित करेंगे(`'X'`, `'O'`, or `null`). हमने पहले से ही Board के constructor में `squares` array को परिभाषित किया है, और हम इसे पढ़ने के लिए Board के `renderSquare` मेथड को संशोधित करेंगे:
 
 ```javascript{2}
   renderSquare(i) {
     return <Square value={this.state.squares[i]} />;
   }
 ```
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/gWWQPY?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/gWWQPY?editors=0010)**
 
-Each Square will now receive a `value` prop that will either be `'X'`, `'O'`, or `null` for empty squares.
+प्रत्येक स्क्वायर को अब एक `value` prop प्राप्त होगा जो या तो खाली वर्गों के लिए `'X'`, `'O'` या `null` होगा।
```

Comment:
should be value` prop प्राप्त होगा जो खाली वर्गों के लिए `'X'`, `'O'` या `null` होगा।